brainstormforce/ultimate-addons-for-gutenberg
# UAG-341 - Fix color not applying to content timeline arrow in editor

### Description

- Card - https://ua-gutenberg.atlassian.net/jira/software/projects/UAG/boards/2/backlog?selectedIssue=UAG-341

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop Content timeline > Select orientation > Left
- Add styling to every content template and check working.

### Checklist:
